### ISSUE TITLE
Feature: Ability to use blog post's feature image on the home page

### ISSUE THREAD

**Issue by stephenajulu** [jake-101/bael-template#16]
Hi Jake. How are you?
This theme is great. Thank you. I hope t use it to transfer my blog from WordPress soon.
But i would like to request a feature, if there's a way to introduce feature images to the homepage.
This feature would be great if it was optional as some including i like the theme they way it is. But a friend(non dev) asked if you had time to implement this.

**Comment by jake-101** [comment]
Hi Stephen, thanks for the nice comments and I am doing pretty well. I haven't checked the issues for awhile but there is some good news for you! I do have that feature implemented as a secondary layout already. It doesn't have a grid layout however, and each image is full screen.

check out https://bael-template-alt-layout.netlify.app/ to see how it looks and https://bael-theme.jake101.com/blog/2019-05-26-0-2-1-update-new-alt-layout/ for how to turn it on.

**Comment by stephenajulu** [comment]
Thank you

**Comment by canvasandcode** [comment]
Hi @jake-101 I'm loving this template and how much work has gone into it. I'm also wanting to change the layout to the alternative version. I have followed your steps to update it in admin, however when Netlify deploys I can't access the site (says page not found). When forcing a deploy in Netlify, I can see in the log the following:

**TypeError: Cannot read property 'length' of undefined
at a.total (components/FullGrid.vue:129:0)**

I switched back to the normal layout and everything works as it should, so I'm thinking it's an issue with the layout?

Thanks in advance!
